frontend/js/utils/locale.js

```diff
@@ -0,0 +1,44 @@
+import { Mandarin } from 'flatpickr/dist/l10n/zh.js'
+import { Russian } from 'flatpickr/dist/l10n/ru.js'
+import { French } from 'flatpickr/dist/l10n/fr.js'
+import { Polish } from 'flatpickr/dist/l10n/pl.js'
+
+export const locales = {
+  en: {
+    'date-fns': require('date-fns/locale/en')
+  },
+  'zh-Hans': {
+    'date-fns': require('date-fns/locale/zh_cn'),
+    flatpickr: Mandarin
+  },
+  ru: {
+    'date-fns': require('date-fns/locale/ru'),
+    flatpickr: Russian
+  },
+  fr: {
+    'date-fns': require('date-fns/locale/fr'),
+    flatpickr: French
+  },
+  pl: {
+    'date-fns': require('date-fns/locale/pl'),
+    flatpickr: Polish
+  }
+}
+
+const twelveHourFormatLocales = ['en']
```

Comment:
This - I guess its currently fine, but I guess in the future we should add another prop to the `_date_picker` for user to override.